client/src/main.rs

```diff
@@ -612,6 +625,31 @@ fn add_peer(interface: &InterfaceName, opts: AddPeerOpts) -> Result<(), Error> {
     Ok(())
 }
 
+fn rename_peer(interface: &InterfaceName, opts: RenamePeerOpts) -> Result<(), Error> {
+    let InterfaceConfig { server, .. } = InterfaceConfig::from_interface(interface)?;
+    let api = Api::new(&server);
+
+    log::info!("Fetching peers");
+    let peers: Vec<Peer> = api.http("GET", "/admin/peers")?;
+
+    if let Some((peer_request, old_name)) = prompts::rename_peer(&peers, &opts)? {
+        log::info!("Renaming peer...");
+
+        let id = peers
+            .iter()
+            .filter(|p| p.name == old_name)
+            .map(|p| p.id)
+            .next()
+            .expect("Peer disappeared");
```

Comment:
Will do. I thought about simply using `unwrap` because I can't think of a way in which this would get triggered unless someone manually edited the sqlite DB while performing this action :stuck_out_tongue_winking_eye: But your change is more consistent for sure!